docs/src/release-notes/v0_17.md

```diff
@@ -14,7 +14,7 @@ The Variables pane in the integrated debugger now supports global variables:
 
 We added a run and debug button above the file editor area when you open a Julia file that makes it easier to run the currently active file:
 
-![run-debugger buttons](./0.17/run_debug_buttons.png)
+![run-debugger buttons](./0.17/run_debug_buttons_cropped.png)
```

Comment:
@davidanthoff if you don't like this cropping, please do reuse the original image :)

Reply:
Perfect, way better than before!